src/constants.js

```diff
@@ -130,3 +131,8 @@ export const IPC_RENDERER = ipcRenderer;
  * Flag to hide/show elements after ledger integration is done
  */
 export const LEDGER_ENABLED = false;
+
+/**
+ * Store to be used in the wallet and all HathorWallet objects created
+ */
+export const STORE = new HybridStore();
```

Comment:
I don't like putting this in the constants file. I feel we should create a `storeSingleton` in the `./storage.js` and import from there. Or we can create a `./storeInstance.js`, for instance.

Reply:
Done!